.test-infra/jenkins/CommonTestProperties.groovy

```diff
@@ -26,7 +26,7 @@ class CommonTestProperties {
   }
 
   static String getFlinkVersion() {
-    return "1.13"
+    return "1.12" //Flink Version in dataproc 2.0
```

Comment:
I don't think this sets the Flink version used in the load tests. I think that's set in each load test config, e.g.

https://github.com/apache/beam/blob/d3f320426a115b8c986a817fe2ba87f9fd7f2192/.test-infra/jenkins/job_LoadTests_GBK_Flink_Go.groovy#L201

Looks like it's already set to 1.12, so you can probably revert this change.